src/main/java/HomePage.java

```diff
@@ -0,0 +1,21 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage {
+
+    WebDriver driver;
+
+    public HomePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public ResultPage enterSearchedobject() {
```

Comment:
Can you convert this function into a generic function like this: `enterSearch("Nintendo Switch")`